terraform/variables.tf

```diff
@@ -121,3 +121,26 @@ variable "tf_script_version" {
   default     = ""
 }
 
+variable "is_rms_private_endpoint_required" {
+  type        = bool
+  description = "Set resource manager private endpoint. Default value is true"
+  default     = true
+}
+
+variable "add_new_rms_private_endpoint" {
+  type        = bool
+  description = "Add new resource manager private endpoint"
+  default     = false
+}
+
+variable "add_existing_rms_private_endpoint" {
```

Comment:
Do we need two different variables: add_new_rms_private_endpoint and add_existing_rms_private_endpoint? Why cant  we use just use_existing_rms_endpoint? If that is false that create new endpoint if rms endpoint is used.

Reply:
I put it as two because users will explicitly know new one will create if option is picked and vice versa

Reply:
I am kind of in favor of one to reduce the number of fields on the UI and since these are checkbox instead of radio buttons it does not show the relation explicitly. The fact that user does not select existing endpoint implies that one has to be created.